packages/core/src/components/ModalNew/footers/ModalFooterWizard/ModalFooterWizard.module.scss

```diff
@@ -0,0 +1,14 @@
+.primary {
+  order: 2;
```

Comment:
Why are you controlling the order?

Reply:
> it is a bad practice, but design wants the primary to be first tabbed order
and using positive tabindex is even a worse practice
so I've manipulated the render order